vme/doc/dil.html

```diff
@@ -2174,21 +2174,23 @@ <H2><A NAME="bfrestore"></A>
 <PRE>
 <hr><a name="bfmelee">
 integer meleeattack ( ch : unitptr, vict : unitptr,
-                     bonus : integer, wtype : integer )
-   ch    : The character which should make an additional attack.
-   vict  : The victim of the attack.
-   bonus : Any penalty or bonus added to the attack.
-   wtype : The weapon type of the attack, if a valid type then that is
-           used for the attack purpose, otherwise the default weapon/hand
-           attack is used.
+                     bonus : integer, wtype : integer, primary : integer )
+   ch     : The character which should make an additional attack.
+   vict   : The victim of the attack.
```

Comment:
This will likely get overwritten by a Wiki sync script. 

Can you edit the Wiki instead please? Feel free to make those entries prettier when you're at it !

Reply:
Done :)